cluster-scope/overlays/nerc-ocp-prod/apiservers/default_patch.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: config.openshift.io/v1
+kind: APIServer
```

Comment:
Looks like we don't have a base yet for the API server in base that's included in overlays/common similar to what was done for the ingresscontroller (e.g. `cluster-scope/base/operator.openshift.io/ingresscontrollers/default`). This is causing the kustomize build to fail for `nerc-ocp-prod` overlay.

Reply:
Thanks @jtriley I'll fix this

Reply:
@chrisstafford I see that this is outdated is it ready for review again?

Reply:
I mentioned this during yesterday's weekly meeting briefly and @larsks mentioned he'd prefer to have the basic API server manifest in base and patch in the config in the overlays as I outlined previously. Mind updating?